distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ParallelConsumer.java

```diff
@@ -0,0 +1,71 @@
+package ddf.common.test.restito;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+/**
+ * Useful for processing Restito responses safely in parallel, with additional effects if desired.
+ */
+public class ParallelConsumer {
```

Comment:
This class will be getting deleted. Forgot I made it. This was back when I was trying to work with blocking calls. You may disregard.